tests/test_wallet.py

```diff
@@ -348,6 +348,17 @@ async def test_get_tokens(self):
     async def test_is_signing_key_set(self):
         assert await self.wallet.is_signing_key_set()
 
+    async def test_toggle_2fa(self):
+        result = await self.wallet.enable_2fa()
+        self.assertTrue(result)
+
+        pub_key_hash = self.wallet.zk_signer.pubkey_hash_str()
+        result = await self.wallet.disable_2fa(pub_key_hash)
+        self.assertTrue(result)
```

Comment:
I think that it would be much better to get the account state and see if it is No2FA

Reply:
I see. Yeah. Will do